configure

```diff
@@ -1407,7 +1407,7 @@ Optional Packages:
   --with-key3[=ARG]       Set the third obfuscation key (ignored unless the
                           other two are also specified)
   --with-maxconn[=ARG]    optionally set the maximum connections the core can
-                          handle (default: 16384) NOT USED YET - EXPERIMENTAL
+                          handle (default: 1024)
```

Comment:
Shouldn't this say that the default is 3072 if epoll is enabled and FD_SETSIZE (usually 1024) otherwise?

Reply:
changed comment, and rebased